hedera-node/src/test/java/com/hedera/services/state/exports/SignedStateBalancesExporterTest.java

```diff
@@ -626,17 +626,14 @@ private String expectedExportDir() {
 	public void initsAsExpected() {
 		// expect:
 		assertEquals(ledgerFloat, subject.expectedFloat);
-		assertEquals(SignedStateBalancesExporter.NEVER, subject.periodEnd);
 	}
 
 	@Test
 	public void exportsWhenPeriodSecsHaveElapsed() {
+		now = subject.periodEnd;
+		anEternityLater = now.plusSeconds(dynamicProperties.balancesExportPeriodSecs() * 2);
 		assertFalse(subject.isTimeToExport(now));
-		assertEquals(now.plusSeconds(dynamicProperties.balancesExportPeriodSecs()), subject.periodEnd);
-		assertFalse(subject.isTimeToExport(shortlyAfter));
-		assertEquals(now.plusSeconds(dynamicProperties.balancesExportPeriodSecs()), subject.periodEnd);
 		assertTrue(subject.isTimeToExport(anEternityLater));
-		assertEquals(anEternityLater.plusSeconds(dynamicProperties.balancesExportPeriodSecs()), subject.periodEnd);
```

Comment:
We should keep the test with `assertFalse(subject.isTimeToExport(shortlyAfter));`, also we should keep the check on whether `periodEnd` got changed or not.

Reply:
Yeah, I will put back similar test cases, though may not be exactly same.

Reply:
I changed the test scenarios to cover those changes.